pkg/script/kubeadm-bootstrap.go

```diff
@@ -0,0 +1,27 @@
+package script
+
+import "bytes"
+
+const kubeadmBootstrapTmpl string = `#!/bin/sh
```

Comment:
Why `sh` and not `bash`?

Reply:
it was that way in the original script... should I change it?